src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -165,7 +165,7 @@ > Print(2)
 > ", runner.Console.Out.ToString());
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/23560")]
```

Comment:
Uh, why....?

Reply:
I'm not sure exactly why the tests are failing - could be that the machines we are testing this branch on have different version of .NET that exposes the linked issue.